src/GraphQL/loaders/defaultGraphQLTypes.js

```diff
@@ -553,7 +554,7 @@ const ACL = new GraphQLObjectType({
         Object.keys(p).forEach(rule => {
           if (rule !== '*' && rule.indexOf('role:') !== 0) {
             users.push({
-              userId: rule,
+              userId: toGlobalId('_User', rule),
```

Comment:
Can we make to work with either Relay global id or Parse object id?

Reply:
Its the output type, we should return only the RelayId, otherwise it would be hard in a near further to get off `objectId`

Reply:
Sorry. My bad. You're right.